synapse/config/registration.py

```diff
@@ -31,6 +31,8 @@ def read_config(self, config):
                 strtobool(str(config["disable_registration"]))
             )
 
+        self.validity_period = config.get("validity_period", 0)
```

Comment:
I would store this duration in milliseconds for consistency with the rest of the code base.